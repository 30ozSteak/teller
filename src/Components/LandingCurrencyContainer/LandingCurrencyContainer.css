.currency-container {
  margin-top: 8rem;
  width: 25rem;
  height: 28rem;
  overflow: scroll;
}
