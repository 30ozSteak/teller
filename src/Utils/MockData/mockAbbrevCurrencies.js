export const mockAbbrevCurrencies = [
  {
    "symbol": "BTC",
    "price": 3586.16,
    "percent_change": 0.83,
    "name": "Bitcoin",
    "rank": "1",
  },
  {
    "symbol": "XRP",
    "price": 0.33,
    "percent_change": 3.94,
    "name": "XRP",
    "rank": "2",
  },
  {
    "symbol": "ETH",
    "price": 95.07,
    "percent_change": 0.20,
    "name": "Ethereum",
    "rank": "3",
  },
  {
    "symbol": "EOS",
    "price": 2.54,
    "percent_change": 6.87,
    "name": "EOS",
    "rank": "4",
  },
  {
    "symbol": "XLM",
    "price": 0.11,
    "percent_change": -0.58,
    "name": "Stellar",
    "rank": "5",
  },
  {
    "symbol": "USDT",
    "price": 1.02,
    "percent_change": -0.37,
    "name": "Tether",
    "rank": "6",
  },
  {
    "symbol": "BCH",
    "price": 100.65,
    "percent_change": 12.32,
    "name": "Bitcoin Cash",
    "rank": "7",
  },
  {
    "symbol": "LTC",
    "price": 28.99,
    "percent_change": -1.97,
    "name": "Litecoin",
    "rank": "8",
  },
  {
    "symbol": "BSV",
    "price": 81.84,
    "percent_change": -1.04,
    "name": "Bitcoin SV",
    "rank": "9",
  },
  {
    "symbol": "TRX",
    "price": 0.01,
    "percent_change": 0.17,
    "name": "TRON",
    "rank": "10",
  }
]