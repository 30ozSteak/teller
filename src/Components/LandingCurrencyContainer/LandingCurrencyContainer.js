import React, { Component } from "react";
import "./LandingCurrencyContainer.css";
import LandingCurrency from "../LandingCurrency/LandingCurrency";

class LandingCurrencyContainer extends Component {
  constructor() {
    super();
    this.state = {
      active: false
    };
  }

  render() {
    return (
      <div className="currency-container">
        <LandingCurrency />
      </div>
    );
  }
}

export default LandingCurrencyContainer;
