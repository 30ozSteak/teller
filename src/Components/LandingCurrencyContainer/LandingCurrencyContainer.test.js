import React from 'react'
import { shallow } from 'enzyme'
import LandingCurrencyContainer from './LandingCurrencyContainer'
import { mockAbbrevCurrencies } from '../../Utils/Mockdata/mockAbbrevCurrencies'

describe('LandingCurrencyContainer Component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<LandingCurrencyContainer
                        abbrevCurrencies={mockAbbrevCurrencies}
                      />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})