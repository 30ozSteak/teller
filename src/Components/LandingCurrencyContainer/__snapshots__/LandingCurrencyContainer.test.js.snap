// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingCurrencyContainer Component should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LandingCurrencyContainer
    abbrevCurrencies={
      Array [
        Object {
          "name": "Bitcoin",
          "percent_change": 0.83,
          "price": 3586.16,
          "rank": "1",
          "symbol": "BTC",
        },
        Object {
          "name": "XRP",
          "percent_change": 3.94,
          "price": 0.33,
          "rank": "2",
          "symbol": "XRP",
        },
        Object {
          "name": "Ethereum",
          "percent_change": 0.2,
          "price": 95.07,
          "rank": "3",
          "symbol": "ETH",
        },
        Object {
          "name": "EOS",
          "percent_change": 6.87,
          "price": 2.54,
          "rank": "4",
          "symbol": "EOS",
        },
        Object {
          "name": "Stellar",
          "percent_change": -0.58,
          "price": 0.11,
          "rank": "5",
          "symbol": "XLM",
        },
        Object {
          "name": "Tether",
          "percent_change": -0.37,
          "price": 1.02,
          "rank": "6",
          "symbol": "USDT",
        },
        Object {
          "name": "Bitcoin Cash",
          "percent_change": 12.32,
          "price": 100.65,
          "rank": "7",
          "symbol": "BCH",
        },
        Object {
          "name": "Litecoin",
          "percent_change": -1.97,
          "price": 28.99,
          "rank": "8",
          "symbol": "LTC",
        },
        Object {
          "name": "Bitcoin SV",
          "percent_change": -1.04,
          "price": 81.84,
          "rank": "9",
          "symbol": "BSV",
        },
        Object {
          "name": "TRON",
          "percent_change": 0.17,
          "price": 0.01,
          "rank": "10",
          "symbol": "TRX",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="currency-container"
      >
        <div
          className="sort"
        >
          <p
            className="active"
          >
            Rank
          </p>
          <p>
            Price
          </p>
          <p>
            %Change
          </p>
        </div>
        <LandingCurrency
          currency={
            Object {
              "name": "Bitcoin",
              "percent_change": 0.83,
              "price": 3586.16,
              "rank": "1",
              "symbol": "BTC",
            }
          }
          index={0}
        />
        <LandingCurrency
          currency={
            Object {
              "name": "XRP",
              "percent_change": 3.94,
              "price": 0.33,
              "rank": "2",
              "symbol": "XRP",
            }
          }
          index={1}
        />
        <LandingCurrency
          currency={
            Object {
              "name": "Ethereum",
              "percent_change": 0.2,
              "price": 95.07,
              "rank": "3",
              "symbol": "ETH",
            }
          }
          index={2}
        />
        <LandingCurrency
          currency={
            Object {
              "name": "EOS",
              "percent_change": 6.87,
              "price": 2.54,
              "rank": "4",
              "symbol": "EOS",
            }
          }
          index={3}
        />
        <LandingCurrency
          currency={
            Object {
              "name": "Stellar",
              "percent_change": -0.58,
              "price": 0.11,
              "rank": "5",
              "symbol": "XLM",
            }
          }
          index={4}
        />
        <LandingCurrency
          currency={
            Object {
              "name": "Tether",
              "percent_change": -0.37,
              "price": 1.02,
              "rank": "6",
              "symbol": "USDT",
            }
          }
          index={5}
        />
        <LandingCurrency
          currency={
            Object {
              "name": "Bitcoin Cash",
              "percent_change": 12.32,
              "price": 100.65,
              "rank": "7",
              "symbol": "BCH",
            }
          }
          index={6}
        />
        <LandingCurrency
          currency={
            Object {
              "name": "Litecoin",
              "percent_change": -1.97,
              "price": 28.99,
              "rank": "8",
              "symbol": "LTC",
            }
          }
          index={7}
        />
        <LandingCurrency
          currency={
            Object {
              "name": "Bitcoin SV",
              "percent_change": -1.04,
              "price": 81.84,
              "rank": "9",
              "symbol": "BSV",
            }
          }
          index={8}
        />
        <LandingCurrency
          currency={
            Object {
              "name": "TRON",
              "percent_change": 0.17,
              "price": 0.01,
              "rank": "10",
              "symbol": "TRX",
            }
          }
          index={9}
        />
        <div
          className="bottom-gradient"
        />
      </div>,
      "className": "sort-box",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="sort"
          >
            <p
              className="active"
            >
              Rank
            </p>
            <p>
              Price
            </p>
            <p>
              %Change
            </p>
          </div>,
          Array [
            <LandingCurrency
              currency={
                Object {
                  "name": "Bitcoin",
                  "percent_change": 0.83,
                  "price": 3586.16,
                  "rank": "1",
                  "symbol": "BTC",
                }
              }
              index={0}
            />,
            <LandingCurrency
              currency={
                Object {
                  "name": "XRP",
                  "percent_change": 3.94,
                  "price": 0.33,
                  "rank": "2",
                  "symbol": "XRP",
                }
              }
              index={1}
            />,
            <LandingCurrency
              currency={
                Object {
                  "name": "Ethereum",
                  "percent_change": 0.2,
                  "price": 95.07,
                  "rank": "3",
                  "symbol": "ETH",
                }
              }
              index={2}
            />,
            <LandingCurrency
              currency={
                Object {
                  "name": "EOS",
                  "percent_change": 6.87,
                  "price": 2.54,
                  "rank": "4",
                  "symbol": "EOS",
                }
              }
              index={3}
            />,
            <LandingCurrency
              currency={
                Object {
                  "name": "Stellar",
                  "percent_change": -0.58,
                  "price": 0.11,
                  "rank": "5",
                  "symbol": "XLM",
                }
              }
              index={4}
            />,
            <LandingCurrency
              currency={
                Object {
                  "name": "Tether",
                  "percent_change": -0.37,
                  "price": 1.02,
                  "rank": "6",
                  "symbol": "USDT",
                }
              }
              index={5}
            />,
            <LandingCurrency
              currency={
                Object {
                  "name": "Bitcoin Cash",
                  "percent_change": 12.32,
                  "price": 100.65,
                  "rank": "7",
                  "symbol": "BCH",
                }
              }
              index={6}
            />,
            <LandingCurrency
              currency={
                Object {
                  "name": "Litecoin",
                  "percent_change": -1.97,
                  "price": 28.99,
                  "rank": "8",
                  "symbol": "LTC",
                }
              }
              index={7}
            />,
            <LandingCurrency
              currency={
                Object {
                  "name": "Bitcoin SV",
                  "percent_change": -1.04,
                  "price": 81.84,
                  "rank": "9",
                  "symbol": "BSV",
                }
              }
              index={8}
            />,
            <LandingCurrency
              currency={
                Object {
                  "name": "TRON",
                  "percent_change": 0.17,
                  "price": 0.01,
                  "rank": "10",
                  "symbol": "TRX",
                }
              }
              index={9}
            />,
          ],
          <div
            className="bottom-gradient"
          />,
        ],
        "className": "currency-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="active"
              >
                Rank
              </p>,
              <p>
                Price
              </p>,
              <p>
                %Change
              </p>,
            ],
            "className": "sort",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Rank",
                "className": "active",
              },
              "ref": null,
              "rendered": "Rank",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Price",
              },
              "ref": null,
              "rendered": "Price",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "%Change",
              },
              "ref": null,
              "rendered": "%Change",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "Bitcoin",
              "percent_change": 0.83,
              "price": 3586.16,
              "rank": "1",
              "symbol": "BTC",
            },
            "index": 0,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "XRP",
              "percent_change": 3.94,
              "price": 0.33,
              "rank": "2",
              "symbol": "XRP",
            },
            "index": 1,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "Ethereum",
              "percent_change": 0.2,
              "price": 95.07,
              "rank": "3",
              "symbol": "ETH",
            },
            "index": 2,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "EOS",
              "percent_change": 6.87,
              "price": 2.54,
              "rank": "4",
              "symbol": "EOS",
            },
            "index": 3,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "Stellar",
              "percent_change": -0.58,
              "price": 0.11,
              "rank": "5",
              "symbol": "XLM",
            },
            "index": 4,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "Tether",
              "percent_change": -0.37,
              "price": 1.02,
              "rank": "6",
              "symbol": "USDT",
            },
            "index": 5,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "Bitcoin Cash",
              "percent_change": 12.32,
              "price": 100.65,
              "rank": "7",
              "symbol": "BCH",
            },
            "index": 6,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "Litecoin",
              "percent_change": -1.97,
              "price": 28.99,
              "rank": "8",
              "symbol": "LTC",
            },
            "index": 7,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "Bitcoin SV",
              "percent_change": -1.04,
              "price": 81.84,
              "rank": "9",
              "symbol": "BSV",
            },
            "index": 8,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "currency": Object {
              "name": "TRON",
              "percent_change": 0.17,
              "price": 0.01,
              "rank": "10",
              "symbol": "TRX",
            },
            "index": 9,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "bottom-gradient",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="currency-container"
        >
          <div
            className="sort"
          >
            <p
              className="active"
            >
              Rank
            </p>
            <p>
              Price
            </p>
            <p>
              %Change
            </p>
          </div>
          <LandingCurrency
            currency={
              Object {
                "name": "Bitcoin",
                "percent_change": 0.83,
                "price": 3586.16,
                "rank": "1",
                "symbol": "BTC",
              }
            }
            index={0}
          />
          <LandingCurrency
            currency={
              Object {
                "name": "XRP",
                "percent_change": 3.94,
                "price": 0.33,
                "rank": "2",
                "symbol": "XRP",
              }
            }
            index={1}
          />
          <LandingCurrency
            currency={
              Object {
                "name": "Ethereum",
                "percent_change": 0.2,
                "price": 95.07,
                "rank": "3",
                "symbol": "ETH",
              }
            }
            index={2}
          />
          <LandingCurrency
            currency={
              Object {
                "name": "EOS",
                "percent_change": 6.87,
                "price": 2.54,
                "rank": "4",
                "symbol": "EOS",
              }
            }
            index={3}
          />
          <LandingCurrency
            currency={
              Object {
                "name": "Stellar",
                "percent_change": -0.58,
                "price": 0.11,
                "rank": "5",
                "symbol": "XLM",
              }
            }
            index={4}
          />
          <LandingCurrency
            currency={
              Object {
                "name": "Tether",
                "percent_change": -0.37,
                "price": 1.02,
                "rank": "6",
                "symbol": "USDT",
              }
            }
            index={5}
          />
          <LandingCurrency
            currency={
              Object {
                "name": "Bitcoin Cash",
                "percent_change": 12.32,
                "price": 100.65,
                "rank": "7",
                "symbol": "BCH",
              }
            }
            index={6}
          />
          <LandingCurrency
            currency={
              Object {
                "name": "Litecoin",
                "percent_change": -1.97,
                "price": 28.99,
                "rank": "8",
                "symbol": "LTC",
              }
            }
            index={7}
          />
          <LandingCurrency
            currency={
              Object {
                "name": "Bitcoin SV",
                "percent_change": -1.04,
                "price": 81.84,
                "rank": "9",
                "symbol": "BSV",
              }
            }
            index={8}
          />
          <LandingCurrency
            currency={
              Object {
                "name": "TRON",
                "percent_change": 0.17,
                "price": 0.01,
                "rank": "10",
                "symbol": "TRX",
              }
            }
            index={9}
          />
          <div
            className="bottom-gradient"
          />
        </div>,
        "className": "sort-box",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="sort"
            >
              <p
                className="active"
              >
                Rank
              </p>
              <p>
                Price
              </p>
              <p>
                %Change
              </p>
            </div>,
            Array [
              <LandingCurrency
                currency={
                  Object {
                    "name": "Bitcoin",
                    "percent_change": 0.83,
                    "price": 3586.16,
                    "rank": "1",
                    "symbol": "BTC",
                  }
                }
                index={0}
              />,
              <LandingCurrency
                currency={
                  Object {
                    "name": "XRP",
                    "percent_change": 3.94,
                    "price": 0.33,
                    "rank": "2",
                    "symbol": "XRP",
                  }
                }
                index={1}
              />,
              <LandingCurrency
                currency={
                  Object {
                    "name": "Ethereum",
                    "percent_change": 0.2,
                    "price": 95.07,
                    "rank": "3",
                    "symbol": "ETH",
                  }
                }
                index={2}
              />,
              <LandingCurrency
                currency={
                  Object {
                    "name": "EOS",
                    "percent_change": 6.87,
                    "price": 2.54,
                    "rank": "4",
                    "symbol": "EOS",
                  }
                }
                index={3}
              />,
              <LandingCurrency
                currency={
                  Object {
                    "name": "Stellar",
                    "percent_change": -0.58,
                    "price": 0.11,
                    "rank": "5",
                    "symbol": "XLM",
                  }
                }
                index={4}
              />,
              <LandingCurrency
                currency={
                  Object {
                    "name": "Tether",
                    "percent_change": -0.37,
                    "price": 1.02,
                    "rank": "6",
                    "symbol": "USDT",
                  }
                }
                index={5}
              />,
              <LandingCurrency
                currency={
                  Object {
                    "name": "Bitcoin Cash",
                    "percent_change": 12.32,
                    "price": 100.65,
                    "rank": "7",
                    "symbol": "BCH",
                  }
                }
                index={6}
              />,
              <LandingCurrency
                currency={
                  Object {
                    "name": "Litecoin",
                    "percent_change": -1.97,
                    "price": 28.99,
                    "rank": "8",
                    "symbol": "LTC",
                  }
                }
                index={7}
              />,
              <LandingCurrency
                currency={
                  Object {
                    "name": "Bitcoin SV",
                    "percent_change": -1.04,
                    "price": 81.84,
                    "rank": "9",
                    "symbol": "BSV",
                  }
                }
                index={8}
              />,
              <LandingCurrency
                currency={
                  Object {
                    "name": "TRON",
                    "percent_change": 0.17,
                    "price": 0.01,
                    "rank": "10",
                    "symbol": "TRX",
                  }
                }
                index={9}
              />,
            ],
            <div
              className="bottom-gradient"
            />,
          ],
          "className": "currency-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <p
                  className="active"
                >
                  Rank
                </p>,
                <p>
                  Price
                </p>,
                <p>
                  %Change
                </p>,
              ],
              "className": "sort",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Rank",
                  "className": "active",
                },
                "ref": null,
                "rendered": "Rank",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Price",
                },
                "ref": null,
                "rendered": "Price",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "%Change",
                },
                "ref": null,
                "rendered": "%Change",
                "type": "p",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "Bitcoin",
                "percent_change": 0.83,
                "price": 3586.16,
                "rank": "1",
                "symbol": "BTC",
              },
              "index": 0,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "XRP",
                "percent_change": 3.94,
                "price": 0.33,
                "rank": "2",
                "symbol": "XRP",
              },
              "index": 1,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "Ethereum",
                "percent_change": 0.2,
                "price": 95.07,
                "rank": "3",
                "symbol": "ETH",
              },
              "index": 2,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "EOS",
                "percent_change": 6.87,
                "price": 2.54,
                "rank": "4",
                "symbol": "EOS",
              },
              "index": 3,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "Stellar",
                "percent_change": -0.58,
                "price": 0.11,
                "rank": "5",
                "symbol": "XLM",
              },
              "index": 4,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "Tether",
                "percent_change": -0.37,
                "price": 1.02,
                "rank": "6",
                "symbol": "USDT",
              },
              "index": 5,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "Bitcoin Cash",
                "percent_change": 12.32,
                "price": 100.65,
                "rank": "7",
                "symbol": "BCH",
              },
              "index": 6,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "Litecoin",
                "percent_change": -1.97,
                "price": 28.99,
                "rank": "8",
                "symbol": "LTC",
              },
              "index": 7,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "Bitcoin SV",
                "percent_change": -1.04,
                "price": 81.84,
                "rank": "9",
                "symbol": "BSV",
              },
              "index": 8,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "currency": Object {
                "name": "TRON",
                "percent_change": 0.17,
                "price": 0.01,
                "rank": "10",
                "symbol": "TRX",
              },
              "index": 9,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "bottom-gradient",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
